### PR TITLE
Blog: CES Reloaded (recap of talks to watch)

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/ces-reloaded/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+date: 2020-10-16
+meta_desc: "Watch popular talks and panels from the Cloud Engineering Summit."
+meta_image: cloud_engineering.png
+authors:
+    - sophia-parafina
+tags:
+    - cloud engineering summit=
+---
+
+Mission accomplished. Putting on a new event is always risky, especially when the calendar is saturated with virtual conferences. The goal of the Cloud Engineering Summit was to bring the level of conversation above specific technologies and take a holistic look into the future of the cloud and you knocked it out of the park! Here are the stats:
```
[the remark this answers is not in the document]
Comment (ericrudder) [reply content/blog/ces-reloaded/index.md:12]: "mission accomplished" is (unfortunately) now a political term ...

File: content/blog/ces-reloaded/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+- 46 industry leaders
+- 17 hours of content
+- 3000+ participants
+- 96 countries
+
+<!--more-->
+
+[Joe Duffy's](https://twitter.com/funcofjoe) keynote kicked off the summit by describing the current state of cloud engineer and what the future holds for cloud engineering. Watch Joe's keynote, [The Future of Cloud Engineering](https://cloudengineering.heysummit.com/talks/welcome-keynote/) to set the stage for the summit.
```
Comment (ericrudder) [inline content/blog/ces-reloaded/index.md:21]: there's something wrong with this sentence.

in addition, i think we should be consistent in Cloud Engineer(ing) casing ... I favor "promoting" it (reasonable folks can differ), however, we should pick a lane :)

File: content/blog/ces-reloaded/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+Following the themes Joe addressed, summit speakers covered a wide array of topics ranging from future architecture, to the impact of architecture on workflows, and to how teams will build the cloud infrastructure in the future. Here are the ones you don't want to miss!
```
Comment (ericrudder) [inline content/blog/ces-reloaded/index.md:23]:
```suggestion
Following the themes Joe addressed, Summit speakers covered a wide array of topics ranging from future architecture, to the impact of architecture on workflows, and to how teams will build the cloud infrastructure in the future. Here are the ones you don't want to miss!
```

File: content/blog/ces-reloaded/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    <a href="https://twitter.com/DivineOps"><img src="sasha-rosenbaum.jpg" alt="Sasha Rosenbaum" class="h-16 md:mx-4 my-4"></a>
+    <a href="https://cloudengineering.heysummit.com/talks/the-microsoft-devops-journey-so-far/"><b>The Microsoft DevOps Journey (so far)</b><br>Sasha Rosenbaum</a>
+</div>
+
+<div class="flex flex-col md:flex-row md:flex-wrap items-center justify-left">
+    <a href="https://twitter.com/clenneville"><img src="christie-lenneville.jpg" alt="Christie Lenneville" class="h-16 md:mx-4 my-4"></a>
+    <a href="https://cloudengineering.heysummit.com/talks/designing-for-remote-teams/"><b>All-Remote Teams at Scale</b><br>Christie Lenneville</a>
+</div>
+
+And don't forget to watch our workshop!
```
Comment (ericrudder) [inline content/blog/ces-reloaded/index.md:101]: i think we can give a little more of a hint on the topic, etc.

File: content/blog/ces-reloaded/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
+---
+title: "Cloud Engineering Summit Reloaded"
+date: 2020-10-16
```
Comment (zchase) [inline content/blog/ces-reloaded/index.md:3]: I think this will need to be updated?

**Review by zchase — APPROVED** [review]
I think it would be better if the talk titles looked like links because it didn't occur to me I could click them until I hovered over the text and I think the date needs to be updated.

File: content/blog/ces-reloaded/index.md (hunk 5 above)
Comment (spara) [reply content/blog/ces-reloaded/index.md:3]: yup, will update

**Pull request merged**